nsis/build-multiuser.nsi

```diff
@@ -49,9 +49,9 @@
 ; Inetc (http://nsis.sourceforge.net/Inetc_plug-in)
 ; MD5 (http://nsis.sourceforge.net/MD5_plugin)
 ;---
-!addplugindir /x86-ansi "..\data\nsis\Plugins\x86-ansi\"
-!addplugindir /x86-unicode "..\data\nsis\Plugins\x86-unicode\"
-!addincludedir "..\data\nsis\Include\"
+!addplugindir /x86-ansi "Plugins\x86-ansi\"
+!addplugindir /x86-unicode "Plugins\x86-unicode\"
+!addincludedir "Include\"
```

Comment:
are you sure this is correct?  shouldn't it be `..\nsis\<etc>`?

Reply:
I changed the location of the script to `nsis` also so relative is no longer one step back.  It is all together now.  This also means the installer ends up in the `nsis` directory when all is done.  I've changed this in the docs also.  It runs fine for me and produces a functional installer as far as I can tell in my win 7 VM.  If anything just check it reads right in the docs.

Reply:
thanks, much better (that file alone is 37% of the current repository size).

I think I still would rather see the `build-win` script in the `scripts` directory, and that after running it the generated executable ends in the `dist` directory: not for nothing the `dist` directory is ignored, it's because it only contains generated files.  store it there, and you won't make the mistake of committing it again.  store a script among all other scripts, and chances are that someone else will find it even without reading the relevant documentation section.

but after all, I can do this myself, when I come to it.